.goreleaser.yml

```diff
@@ -72,14 +73,48 @@ signs:
     signature: ${artifact}.sig
     artifacts: none
 dockers:
-  -
-    ids:
+  - ids:
       - centrifugo
     goos: linux
     goarch: amd64
     goarm: ''
     image_templates:
-    - "centrifugo/centrifugo:{{ .Tag }}"
-    - "centrifugo/centrifugo:v{{ .Major }}"
-    - "centrifugo/centrifugo:v{{ .Major }}.{{ .Minor }}"
-    - "centrifugo/centrifugo:latest"
+    - "centrifugo/centrifugo:{{ .Tag }}-amd64"
+    - "centrifugo/centrifugo:v{{ .Major }}-amd64"
+    - "centrifugo/centrifugo:v{{ .Major }}.{{ .Minor }}-amd64"
+    - "centrifugo/centrifugo:latest-amd64"
+    use_buildx: true
+    build_flag_templates:
+    - "--platform=linux/amd64"
+    - "--load"
+  - ids:
+      - centrifugo
+    goos: linux
+    goarch: arm64
+    goarm: ''
+    image_templates:
+    - "centrifugo/centrifugo:{{ .Tag }}-arm64"
+    - "centrifugo/centrifugo:v{{ .Major }}-arm64"
+    - "centrifugo/centrifugo:v{{ .Major }}.{{ .Minor }}-arm64"
+    - "centrifugo/centrifugo:latest-arm64"
+    use_buildx: true
+    build_flag_templates:
+    - "--platform=linux/arm64"
```

Comment:
Let's be more explicit here and use `--platform=linux/arm64/v8`. Also use image templates with `-arm64v8` suffix - like in this config for example: https://github.com/caarlos0/domain_exporter/blob/master/.goreleaser.yml